models/checkout.js

```diff
@@ -0,0 +1,129 @@
+const { query } = require('../db/index');
+
+const Tool = require('./tool.js')
+
+exports.all = async () => {
+  const checkouts = (await query(
+    `SELECT
+      checkouts.id,
+      checkouts."patronName",
+      checkouts."patronContact",
+      checkouts."dueDate",
+      checkouts."toolId",
+      tools."name" as "toolName"
```

Comment:
Serializers probably need to happen in all models where there are joining of tables stuff... ?

Reply:
later, put in "maybe" in trello